product-service/handlers/getProductsById/index.ts

```diff
@@ -0,0 +1,97 @@
+import "source-map-support/register";
+import productList from "../../data/productList.json";
+import { withCorsHeaders } from "../../utils/withCorsHeaders";
+import fetch from "node-fetch";
+
+const RANDOM_NUMBER_API =
+  "http://www.randomnumberapi.com/api/v1.0/random?min=1&max=100";
+
+/**
+ * @swagger
+ * definitions:
+ *   Product:
+ *     required:
+ *       - count
+ *       - id
+ *       - price
+ *       - title
+ *       - src
+ *     properties:
+ *       count:
+ *          type: number
+ *       description:
+ *          type: string
+ *       id:
+ *         type: string
+ *       price:
+ *          type: number
+ *       title:
+ *         type: string
+ *       src:
+ *         type: string
+ *   Error:
+ *     required:
+ *       - message
+ *     properties:
+ *       message:
+ *         type: string
+ */
+
+/**
+ * @swagger
+ *
+ * /products/{id}:
+ *   get:
+ *     description: Get product
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: id
+ *         description: Product id
+ *         required: true
+ *         type: string
+ *         default: 7567ec4b-b10c-48c5-9345-fc73c48a80aa
+ *         in: path
+ *     responses:
+ *       200:
+ *         description: OK
+ *         schema:
+ *           $ref: "#/definitions/Product"
+ *       404:
+ *         description: A product with the specified ID was not found
+ *         schema:
+ *           $ref: "#/definitions/Error"
+ *       500:
+ *         description: Something went wrong. Please try again later
+ *         schema:
+ *            $ref: "#/definitions/Error"
+ */
+export const getProductsById = async (event) => {
+  try {
+    const product = productList.find(
+      ({ id }) => id === event.pathParameters.id
```

Comment:
`event.pathParameters` can be null, so you need to handle this case. Like this:
```js
event.pathParameters?.id
```

Reply:
I've already handled it via try-catch block